src/Tribe/Admin/Settings.php

```diff
@@ -202,6 +216,9 @@ public function add_admin_pages() {
 			]
 		);
 
+		// Redirects users from the outdated Help page to the new Help Hub page if accessed.
```

Comment:
```suggestion
		// Redirects users from the outdated Help page to the new Help Hub page if accessed.
```
I don't have the full context but if the `$original_help_slug` is the actual outdated one then we should not call it original, that may confuse devs in future. Let's call it either classic or old instead.

Reply:
Good point, I switched to `old_`